src/units.cpp

```diff
@@ -358,20 +366,33 @@ void Units::getUnit(size_t index, std::string &reference, std::string &prefix, d
     }
 }
```

Comment:
I'm not doing anything about that, there is a lot of order mismatch throughout the code.  I don't think it is anything to spend time on.